src/main/java/edu/harvard/iq/dataverse/util/SystemConfig.java

```diff
@@ -1,6 +1,8 @@
 package edu.harvard.iq.dataverse.util;
 
 import com.ocpsoft.pretty.PrettyContext;
+import com.rometools.utils.Strings;
```

Comment:
This seems like a new dependency on rometools. Do we need it?

Reply:
Good catch - it's gone.